src/pages/blog/ce-qui-caractérise-le-top-20%-des-devs.md

```diff
@@ -0,0 +1,97 @@
+
+## Mais il n’y a pas que la technique pour faire partir du top 20%
+
+### Travail en équipe
+
+On a assez parler technique là, car même pour chez les devs où c’est un sujet très important (et à juste titre faut le dire), il n’y a pas que la technique.
+
+Les meilleurs développeurs savent comment communiquer efficacement avec les différentes parties prenantes, qu'il s'agisse des clients/utilisateurs, d’autres collègues développeurs, de chefs de projet, ou de designers, etc. Ils sont capables de recevoir et de donner des feedbacks constructifs, et contribuent à la dynamique positive de l'équipe.
+
+Si vous en doutez regarder autour de vous la grande majorité des logiciels qui marchent réellement dont les gens veulent et qui résoudent des problèmes puis demander vous s’ils auraient eu la même portée coder par un excellent dev certes mais tout seul dans sa grotte ou mal accompagné.
+
+Votre compétence marginale aussi impressionnante soit-elle trouvera ses limites face à une équipe même moyenne mais soudée et qui travail correctement en équipe. Le marché tranchera et cela peut faire mal.
+
+### Focus sur l'impact
+
+Ils comprennent l'importance de l'impact de leur travail sur le produit final et sur l'utilisateur/client. Ils sont motivés par la création de valeur réelle, plutôt que par la simple écriture de code.
+
+C’est un point auquel, je suis particulièrement sensible. Au début, on passe quasi tous par la phase “pisseurs de code” comme on dit dans le jargon. Personnellement, je ne la vois pas systématiquement que d’un mauvais œil, c’est une partie intégrante d’une phase de progression, l’erreur serait d’y rester pire de penser que plus de code impressionnant on envoi, plus on est excellente.
+
+Une phrase de devs seniors avec qui j’ai échangé et qui m’a beaucoup marqué est la suivante :
+
+> “La meilleure ligne de code est celle que l’on n'écrit pas.”
+
+### Pragmatisme
+
+Les développeurs du top 20% sont pragmatiques. Ils savent faire la part des choses entre perfection et productivité. Ils comprennent quand il est nécessaire de faire des compromis pour avancer plus rapidement sans sacrifier la qualité du produit.
+
+C’est certainement la caractéristique sur laquelle j’aurais à le plus fournir d’effort. Etant à l’extrême perfectionniste, je vois littéralement comment cela est handicapant au quotidien. Si vous voulez faire partir des meilleurs vous devrez entrainer ce “soft skill” qui changera votre vie d’entrepreneur, de collaborateur, de freelance, etc. Pensez y vraiment, c’est dur mais nécessaire pour faire partir de la crème.
```

Comment:
Do you think you could give an example? Ideally something you've personally done.

Reply:
Done. 🙂